Update INSTALL.md

GisUsers opened the pull request:
## Description

On Fedora 35 Workstation (fresh install),  it is not possible to install dependencies following the instructions provided.
2 packages have no match and some others are required but not included (python3-sip-devel protobuf-lite protobuf-lite-devel libzstd-devel qt5-qtserialport-devel).
Assuming the currently displayed command worked on previous Fedora versions (which ones?), I propose to keep it for reference, renaming the title: "Install command for dependencies (older versions)" while providing an updated "Install command for dependencies (Fedora 35 Workstation)" that correctly installs all required packages with the latest distribution, allowing to successfully compile.
I've tested the updated command multiple times with success (but of course a double check is welcome).

GisUsers committed:
Update INSTALL.md

On Fedora 35 Workstation (fresh install),  it is not possible to install dependencies following the instructions provided.
2 packages have no match and some others are required but not included (python3-sip-devel protobuf-lite protobuf-lite-devel libzstd-devel qt5-qtserialport-devel).
Assuming the currently displayed command worked on previous Fedora versions (which ones?), I propose to keep it for reference, renaming the title: "Install command for dependencies (older versions)" while providing an updated "Install command for dependencies (Fedora 35 Workstation)" that correctly installs all required packages with the latest distribution, allowing to successfully compile.
I've tested the updated command multiple times with success (but of course a double check is welcome).

DelazJ commented:
@GisUsers instead of two separate tables, can you instead create a single one with two columns on the model of what's done for debian-based distributions (section 3.3)? And place the most recent version on top.

GisUsers committed:
Update INSTALL.md

Created properly formatted table for Fedora install build dependencies as suggested.
